gcp_variant_transforms/libs/bigquery_vcf_data_converter_test.py

```diff
@@ -701,6 +701,58 @@ def test_get_variant_info(self):
     self.assertEqual(expected_variant_info,
                      self._variant_generator._get_variant_info(row))
 
+  def test_get_variant_info_annotation(self):
+    variant_generator = bigquery_vcf_data_converter.VariantGenerator({
+        'CSQ': ['Consequence', 'AF', 'IMPACT']
+    })
+    row = {
+        unicode(ColumnKeyConstants.ALTERNATE_BASES): [
```

Comment:
Is there a way to avoid having these `unicode` casts? The reason is that `unicode` is not supported in python3, so it makes the transition to py3 easier if we don't have these everywhere here.

Reply:
Well, we don't have to use unicode for the unit tests, we can definitely make it to string. I used them here to emphasize that these keys and values are unicode when we loaded from BigQuery, we may have issues if we use `str()` to convert it. Do you prefer to use string instead?